wp-content/themes/core/routes/unsupported_browser/Unsupported_Browser_Controller.php

```diff
@@ -29,6 +29,14 @@ public function get_legacy_image_url( string $filename ): string {
 		return esc_url( trailingslashit( get_template_directory_uri() ) . 'assets/img/theme/legacy-browser/' . $filename );
 	}
 
+	public function get_favicon_url(): string {
+		if ( get_site_icon_url() ) {
+			return esc_url( get_site_icon_url() );
```

Comment:
This function returns a string but it can be filtered so you can just replace this whole method with `return (string) get_site_icon_url();` The `if` conditional isn't needed

Reply:
If we're going to ue WP's icons, we should probably just render `wp_site_icon()` in the template directly. IIRC, using `get_site_icon_url()` will only a single URL for the original site icon which probably isn't compatible with old browsers. We want to make sure we're serving at least the 32px .png file for old browsers.